test/session_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule SessionTest do
+  use ExUnit.Case
+
+  @session_id 1
+  @default_connection_id 0
+
+  setup do
+    # Fake transport will send back any message received to given pid
+    {:ok, connection} =
+      Janus.Connection.start_link(
+        FakeTransport,
+        {@default_connection_id, self()},
```

Comment:
I think you could extract this to varable and this whole call could be collapsed.